.github/workflows/stage.yml

```diff
@@ -84,6 +84,16 @@ jobs:
             -e push=yes \
             -e awx_official=yes
 
+      - name: Log in to Quay
+        run: |
+          echo ${{ secrets.QUAY_TOKEN }} | docker login quay.io -u ${{ secrets.QUAY_USER }} --password-stdin
+
+      - name: tag awx-ee:latest with release tag
+        run: |
+          docker pull quay.io/ansible/awx-ee:latest
+          docker tag quay.io/ansible/awx-ee:latest quay.io/ansible/awx-ee:${{ github.event.release.tag_name }}
+          docker push quay.io/ansible/awx-ee:${{ github.event.release.tag_name }}
```

Comment:
We shouldn't be pushing straight to quay in the stage workflow. There's already a pattern in place where we use ghcr.io as a staging area, run tests, then if everything succeeds we pull the staged images and _then_ push to quay.